src/nixos/machines/ignucius/config/suspend-then-hibernate.nix

```diff
@@ -0,0 +1,19 @@
+{ config, pkgs, ... }:
+
+# Module that implements suspend-then-hibernate for IGNUCIUS
+
+let
+	hibernateSeconds = "60";
+in {
+	services.logind = {
+		# NOTE(Krey): Temporary until I get a new palmrest with it's magnet
+		powerKey = "suspend-then-hibernate"; # Panic Action
```

Comment:
Suspend-then-hibernate doesn't work right now as the device fails hibernation -> Investigate

Reply:
Fixed through kernel management